runtime/environment-cpu.yml

```diff
@@ -7,11 +7,14 @@ dependencies:
   - graph-tool=2.45
   - imbalanced-learn=0.10.0
   - jax=0.3.20
+  - fuzzywuzzy=0.18.0
+  - gensim=3.8.3
   - keras=2.10.0
   - lightgbm=3.1.1
   - loguru=0.6.0
   - mkl=2022.1.0
   - networkx=2.8.8
+  - nltk=3.8.1
```

Comment:
Can you explain how you plan to use some of these natural language processing libraries? There is very limited text data in this challenge. Are you using this to process text, or you planning to represent other types of data as if they were text and using NLP algorithms?

Reply:
We have obtained some success in generating a corpus (without providing external data) in our projects and we'll be using it in context to improve our predictions for this challenge as well, while exploring to see if  NLP'esque approaches are viable for privacy protecting models.